src/parser_data.h

```diff
@@ -156,6 +156,11 @@ struct ly_in;
                                                  revision. */
 /** @} dataparseroptions */
 
+/**
```

Comment:
it should be part of the `dataparseroptions` group, shouldn't it?

Reply:
Well, it was not public before and I moved it here only so that the bits and their mask are together and one is not modified without the other. But it is not really needed to be used so that is why I put it separately. I do not care either way, it can be a part of the options.

Reply:
I would put them together, otherwise it is difficult to find it in doxygen docs

Reply:
Fixed.